packages/cli/src/cli/github/make.cog.pr.ts

```diff
@@ -23,6 +23,8 @@ export class MakeCogGithub extends Github {
    * Format the config files by prettier
    */
   formatConfigFile(path = './config/'): void {
+    this.logger.info({ repository: this.repo }, 'GitHub: Npm Install');
+    execFileSync('npm', ['install', '--include=dev'], { cwd: this.repoName });
```

Comment:
this should only happen once, not everytime this is run.

Reply:
We will cal this `formatConfigFile` function only once every time we run this cli. But I can split this into a seperate function after clone the basemaps-confg repo.

Reply:
that would leave a gotcha in this code path, you could cache it with a simple promise

```typescript
if (this._formatInstall) await this._formatInstall
else this._formatInstall = Promise.resolve().then(f => execFileSync('npm', ['install', '--include=dev'], { cwd: this.repoName })
```

although this is all sync code, so even a `this._formatInstalled = true` would do the trick

Reply:
Got this updated to use `this._formatInstalled = true`. Thanks.